Bottom: dff42226016a574efee33a7fe6ac3f2661c244c6
Top:    afe0164c35f686f6ff94194e14247b1dfce78fcc
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 16:14:48 +0100

Refresh of hid-gtk-drop-in-gl-renderer-wo

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 25a36d8..94a2903 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -741,11 +741,6 @@ ghid_show_crosshair (gboolean paint_new_location)
   if (!paint_new_location)
     return;
 
-  if (gport->x_crosshair < 0 || ghidgui->creating) {// || !gport->has_entered) {
-    printf ("Returning\n");
-    return;
-  }
-
   if (!done_once)
     {
       done_once = 1;
